zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/BackupFileInfo.java

```diff
@@ -88,8 +86,7 @@ public BackupFileInfo(File backedupFile, long modificationTime, long size) {
    * @return the value of the requested zxid part
    */
   public long getZxid(ZxidPart whichZxid) {
-    return whichZxid == ZxidPart.MIN_ZXID
-        ? this.zxidRange.lowerEndpoint()
+    return whichZxid == ZxidPart.MIN_ZXID ? this.zxidRange.lowerEndpoint()
```

Comment:
This file has several style changes. Should we perhaps not include that change in this PR?